script.js

```diff
@@ -227,6 +239,15 @@ themes.forEach((theme) =>
 
 const urlParams = new URLSearchParams(window.location.search);
 
+const searchInput = document.getElementById("search-input");
+searchInput.addEventListener("input", (event) => {
+  populateLibraryList(event.target.value);
+});
+
+document.documentElement.addEventListener("keypress", () => {
+  searchInput.focus();
+});
```

Comment:
```suggestion
document.documentElement.addEventListener("keypress", (event) => {
  if (
    !event.altKey &&
    !event.ctrlKey &&
    !event.metaKey &&
    /^[a-z0-9]$/i.test(event.key)
  ) {
    if (searchInput !== document.activeElement) {
      searchInput.select().focus();
    }
  }
});
```

Above only considers alphanumeric characters & selects the input so that you can run new searches easily without having to first doing `select-all`.